modules/drivers/oracle/test/metabase/test/data/oracle.clj

```diff
@@ -305,3 +306,17 @@
                (into #{}
                      (map :name)
                      (:tables (driver/describe-database :oracle (mt/db))))))))))
+
+(defmethod tx/drop-view! :oracle
+  [driver database view-name materialized?]
+  (let [database-name (get-in database [:settings :database-source-dataset-name])
+        qualified-view (sql.tx/qualify-and-quote driver database-name (name view-name))]
+    (u/ignore-exceptions
```

Comment:
What about detecting if the the error is that the view to be dropped does not exist? E.g., checking if it exists in the exception handler and swallowing the exception only if it doesn't?